### PR TITLE
Add JS array method pop

### PR DESCRIPTION
**Pull request by robgmerrill**
Add JS array method pop

### CODE REVIEW

File: content/javascript/concepts/array-methods/pop.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+Title: '.pop()'
```
Comment (sonnynomnom) [inline content/javascript/concepts/array-methods/pop.md:2]: Double quotes in the metadata!

File: content/javascript/concepts/array-methods/pop.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```js
+array.pop();
+```
+
+## Examples
+
+To pop the element `'sleep'` from the `schedule` array:
+
+```js
+const schedule = ['wake up', 'eat', 'work', 'play', 'sleep'];
```
Comment (sonnynomnom) [inline content/javascript/concepts/array-methods/pop.md:29]: Can we do this?

```js
const schedule = ['wake up', 'eat', 'work', 'play', 'sleep'];
const lastThingToDo = schedule.pop();

console.log(lastThingToDo);
// Output: sleep

console.log(schedule);
// Output: [ 'wake up', 'eat', 'work', 'play' ]
```

- Added line spaces
- Took out the quotes in "// Output: sleep"
- Took out the semicolon in the last line.

**Review by sonnynomnom — COMMENTED** [review]
Left two feedback! Feel free to make the changes and I'll merge!

**Review by robgmerrill — COMMENTED** [review]
Looks good to me! I'll make changes then do another PR

**Comment by sonnynomnom** [comment]
Congrats on your first PR @robgmerrill. Merging!

**Pull request merged**